src/components/SendFlow/Beacon/useSignWithBeacon.tsx

```diff
@@ -10,13 +10,14 @@ import { ImplicitOperations } from "../../../types/AccountOperations";
 import { WalletClient } from "../../../utils/beacon/WalletClient";
 import { useFindNetwork } from "../../../utils/hooks/networkHooks";
 import { useAsyncActionHandler } from "../../../utils/hooks/useAsyncActionHandler";
-import { executeOperations } from "../../../utils/tezos";
+import { Estimation, executeOperations } from "../../../utils/tezos";
 import { DynamicModalContext } from "../../DynamicModal";
 import { SuccessStep } from "../SuccessStep";
 
 export const useSignWithBeacon = (
   operation: ImplicitOperations,
-  message: OperationRequestOutput
+  message: OperationRequestOutput,
+  executeParams?: Estimation
```

Comment:
This should not be an optional parameter since we now always provide the values, same applies to the `executeParams`